repos/fdbt-site/src/pages/api/selectCaps.ts

```diff
@@ -47,19 +46,42 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             return;
         }
 
-        if (!Number.isInteger(Number(cap))) {
+        if (capChoice === 'yes' && !Number.isInteger(Number(cap))) {
             throw Error(`Received invalid cap id: ${cap}`);
         }
 
-        const selectedCap = await getCapContent(Number(cap), noc);
+        const selectedCap = capChoice === 'yes' ? await getCapContent(Number(cap), noc) : undefined;
+        const ticket = getSessionAttribute(req, MATCHING_JSON_ATTRIBUTE);
+        const matchingJsonMetaData = getSessionAttribute(req, MATCHING_JSON_META_DATA_ATTRIBUTE);
```

Comment:
If we move these up to the top of the file, we can keep the `if (capChoice === 'no') {` check and just include `&& !(ticket && matchingJsonMetaData)`